samples/test/.eslintrc.yml

```diff
@@ -2,4 +2,5 @@
 env:
   mocha: true
 rules:
+  node/no-unpublished-require: off
```

Comment:
Generally, we want to move away from using this.  Best way - make sure the `files` section in your `package.json` accurately notes which files are part of the final package

Reply:
@JustinBeckwith do you have a good example of a repo where we do this right that Noah could refer to?

Reply:
Inside of `samples/package.json` you can add a clause like this:
```js
"files": [
  "!test/"
],
```